src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -178,19 +160,22 @@ public void shootFromWall() {
 
   public void idle() {
     // stop gate
-    // slow shooter wheel
+    // stop shooter wheel
     shooterGate.stopMotor();
     leader_shooterMAXRight.stopMotor();
   }
 
   public void purge() {
-    // stow hood
     // reverse gate
     // reverse shooter wheel
     leader_shooterMAXRight.set(-0.2);
     shooterGate.set(-1 * MoPrefs.getShooterGateSetpoint());
```

Comment:
Why is this -1 here?  Could be just a - in front of MoPrefs...

Reply:
I believe Jordan mentioned earlier that a "-1 *" is far more readable than just a "-".  I can definitely see myself completely missing that character in a refactor scenario.